src/components/Carrousel/Carrousel.tsx

```diff
@@ -31,44 +59,27 @@ const useStyles = makeStyles(theme => ({
 
 const Carrousel: React.FC<{ newbies: Newbie[] }> = ({ newbies }) => {
```

Comment:
If now we want to take different approach than using `Carrousel` this component if is really needed should have different name. We want a `List` for mobile and a `Grid` for desktop